librz/io/p/io_srec.c

```diff
@@ -0,0 +1,802 @@
+// Copyright: Manolache183 <manolache.alexandru8@gmail.com>
```

Comment:
Please also add `// SPDX-License-Identifier: LGPL-3.0-only` (if you do not want to use LGPL, please choose the appropriate one).

Reply:
The copyright should be also in the SPDX format.